crates/utils/re_video/src/demux/mod.rs

```diff
@@ -509,6 +507,43 @@ impl VideoDataDescription {
         self.samples.num_elements()
     }
 
+    /// Duration of the video.
+    ///
+    /// For videos that may still grow this is an estimate of the video length so far.
+    /// (we don't know for sure how long the last sample is going to be)
```

Comment:
How does this function respond to dopping of the oldest samples? Does it still count those?